skxray/core.py

```diff
@@ -1226,3 +1226,44 @@ def multi_tau_lags(multitau_levels, multitau_channels):
     lag_steps = np.append(lag_steps, np.array(lag))
 
     return tot_channels, lag_steps
+
+
+def geometric_series(common_ratio, number_of_images, first_term=1):
```

Comment:
@sameera2004 what is the purpose of this function? It does not appear anywhere else in this PR except for its test